content/docs/tasks/traffic-management/app-health-check/index.md

```diff
@@ -42,12 +52,15 @@ NAME                             READY     STATUS    RESTARTS   AGE
 liveness-6857c8775f-zdv9r        2/2       Running   0           1m
 {{< /text >}}
 
-The number '0' in the 'RESTARTS' column means liveness probes worked fine. Readiness probes work in the same way and you can modify `liveness-command.yaml` accordingly to try it yourself.
+The number '0' in the 'RESTARTS' column means liveness probes worked fine.
+Readiness probes work in the same way and you can modify
```

Comment:
Please be explicit. Add the exact command for readiness probes just as it was done for liveness.
Consider using numbered steps. I know that whether you check one or the other first is inconsequential but numbered steps encourage readers to perform both.

Reply:
done